### PR TITLE
Delete calendar export files from server after download

### PR DESCRIPTION
**Pull request by vaxr**
Issue #43

### CODE REVIEW

**Comment by phansch** [comment]
Thanks for working on this! Things are looking fine.
It seems there are some problems with the Travis build, which are not related to your changes. Will take a look at this possibly later today.

File: spec/controllers/calendars_controller_spec.rb [hunk 1]
```diff
@@ -59,6 +59,20 @@
     end
   end
 
+  describe "Calendar export" do
```
Comment (phansch) [inline spec/controllers/calendars_controller_spec.rb]: Not sure if we need this describe section here. I'd prefer to have this test inside the "GET download" block as you are testing the download action.

**Comment by phansch** [comment]
Hey @vaxr, just a quick update on the failing build situation. We figured out what the problem with the build is, but it will not be fixed today. This will take a bit longer, since every external pull request will have the same problem. So this requires some thought first.

Travis has some restrictions on secure environment variables, which currently prevents all external pull requests from runnig properly. (There's more information [here](http://docs.travis-ci.com/user/pull-requests/) if you're interested)

**Comment by vaxr** [comment]
Thanks @phansch .
Maybe you could add me to the BatchZero group so I can push commits directly?

**Comment by phansch** [comment]
Hey @vaxr,

the build issues are fixed now. In order to make the build for this pull request pass, you will have to get the latest changes from upstream first. Then you will have to merge them into your fork and update the pull request again. If you need help with that just let us know.

I also added you to the organization, so you can now push commits directly.
If you start working on an issue, it's best to let us know by leaving a comment on the issue.
Also, please make sure you have read our [conventions](https://github.com/BatchZero/webcal/wiki/Conventions).

**Comment by vaxr** [comment]
Updated pull request. Sorry for the long silence.
Thanks for adding me to the group BTW, although it seems I still don't have write access to this repo.

**Comment by phansch** [comment]
Thanks for the update! I will take a look later today, since I'm a bit out of touch with the webcal :)

**Pull request merged**